src/components/screens/vacation-requests-screen.tsx

```diff
@@ -105,15 +112,22 @@ const VacationRequestsScreen = () => {
   const fetchVacationsRequests = async () => {
```

Comment:
Also the useMemo for this function in this screen and also for the vacations card has the dependency userProfile, but does it actually need this dependency, could it not just be and empty array to run on initial render?

Reply:
Yeah, seems to not need that dependency. Removed it.